linkerd/app/inbound/src/policy/authorize/http.rs

```diff
@@ -92,11 +92,13 @@ where
     fn call(&mut self, req: Req) -> Self::Future {
         match self.policy.check_authorized(self.client_addr, &self.tls) {
             Ok(permit) => {
+                tracing::debug!(?permit, tls = ?self.tls, client = %self.client_addr, "Request authorized");
                 self.metrics.allow(&permit);
                 let svc = self.inner.new_service((permit, self.target.clone()));
                 future::Either::Left(svc.oneshot(req).err_into::<Error>())
             }
             Err(e) => {
+                tracing::info!(tls = ?self.tls, client = %self.client_addr, "Request denied");
```

Comment:
should this include the policy's server label like the TCP events do, or does that come from somewhere else?